src/browser/squirrel-update.coffee

```diff
@@ -231,19 +129,24 @@ exports.handleStartupEvent = (app, squirrelCommand) ->
   switch squirrelCommand
     when '--squirrel-install'
       createShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
           addCommandsToPath ->
             app.quit()
       true
     when '--squirrel-updated'
       updateShortcuts ->
-        installContextMenu ->
+        contextMenuWasEnabled = atom.config.get('core.showAtomInShellContextMenu')
```

Comment:
Tests run inside the render process where globals like this one are available. In the main browser process though where this will run, `atom.config` is not available.